website/addons/s3/static/tests/s3NodeConfig.test.js

```diff
@@ -92,7 +144,7 @@ describe('s3NodeConfigViewModel', () => {
                                 });
                         });
                         it('disables settings in User dosen\'t have auth and is not auth owner', (done) => {
-                            var vm = new s3NodeConfigVM('/api/v1/12345/s3/settings/', '', '/12345');
+                            var vm = new s3NodeConfigVM('', s3ViewModelSettings);
```

Comment:
Why do these settings need to be passed in every test? Can't we just use the defaults?